doc/source/ray-air/deployment.rst

```diff
@@ -0,0 +1,60 @@
+.. _air-deployment:
+
+AIR Deployment Guide
+====================
+
+Here, we describe how you might use or deploy AIR in your infrastructure. There are two main deployment patterns -- a la carte and integration with existing platform.
+
+The core idea is that AIR can be **complementary** to your existing infrastructure and integration tools.
+
+Design Principles
+-----------------
+
+* Ray AIR handles the heavyweight compute aspects of AI apps and services.
+* Ray AIR relies on external integrations (e.g., Tecton, MLFlow, W&B) for Storage and Tracking.
+* Workflow Orchestrators (e.g., AirFlow) are an optional component that can be used for scheduling recurring jobs, launching new Ray clusters for jobs, and running non-Ray compute steps.
+* Lightweight orchestration of task graphs within a single Ray AIR app can be handled with Ray Workflows.
+* Ray AIR libraries can be used independently, within an existing ML platform, or to build a Ray-native ML platform.
+
+
+Ray AIR libraries à la carte
```

Comment:
let's be clear first, witty later. not sure many users will understand this term. maybe "Pick the libraries you need"